src/tests/test_blindsig.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""
+Test for ECC blind signatures
+"""
+import os
+import unittest
+
+from src.pyelliptic.eccblind import ECCBlind
+
+
+class TestBlindSig(unittest.TestCase):
+    """
+    Test case for ECC blind signature
+    """
+    @staticmethod
+    def test_blind_sig():
+        """
+        Perform a test of full sequence using a random certifier's key and a
```

Comment:
If you write docstring in one line it will be shown in the test results. e.g.
```python
    def test_blind_sig(self):
        """Test full sequence using a random certifier key and a random msg"""
```